app/_hub/kong-inc/graphql-rate-limiting-advanced/index.md

```diff
@@ -0,0 +1,482 @@
+        entirely and only stores counters in node memory. A value greater than
+        0 will sync the counters in that many number of seconds.
+    - name: namespace
+      required: false
+      default: random string
+      value_in_examples:
+      description: |
+        The rate limiting library namespace to use for this plugin instance. Counter data and sync configuration is shared in a namespace.
+    - name: strategy
+      required:
+      default: cluster
+      value_in_examples:
+      description: |
+        The sync strategy to use; `cluster` and `redis` are supported
+    - name: redis.host
+      required: semi
+      default:
+      value_in_examples:
+      description: |
+        Host to use for Redis connection when the `redis` strategy is defined
+    - name: redis.port
+      required: semi
+      default:
+      value_in_examples:
+      description: |
+        Port to use for Redis connection when the `redis` strategy is defined
+    - name: redis.timeout
+      required: semi
+      default: 2000
+      value_in_examples:
+      description: |
+        Connection timeout (in milliseconds) to use for Redis connection when the `redis` strategy is defined
+    - name: redis.password
+      required: semi
+      default:
+      value_in_examples:
+      description: |
+        Password to use for Redis connection when the `redis` strategy is defined. If undefined, no AUTH commands are sent to Redis.
+    - name: redis.database
+      required: semi
+      default: 0
+      value_in_examples:
+      description: |
+        Database to use for Redis connection when the `redis` strategy is defined
+    - name: redis.sentinel_master
+      required: semi
+      default:
+      value_in_examples:
+      description: |
+        Sentinel master to use for Redis connection when the `redis` strategy is defined. Defining this value implies using Redis Sentinel.
+    - name: redis.sentinel_role
+      required: semi
+      default:
+      value_in_examples:
+      description: |
+        Sentinel role to use for Redis connection when the `redis` strategy is defined. Defining this value implies using Redis Sentinel.
+    - name: redis.sentinel_addresses
+      required: semi
+      default:
+      value_in_examples:
+      description: |
+        Sentinel addresses to use for Redis connection when the `redis` strategy is defined. Defining this value implies using Redis Sentinel.
+    - name: redis.cluster_addresses
+      required: semi
+      default:
+      value_in_examples:
+      description: |
+        Cluster addresses to use for Redis connection when the `redis` strategy is defined. Defining this value implies using Redis cluster.
+    - name: window_type
+      required:
+      default: sliding
+      value_in_examples:
+      description: |
+        This sets the time window to either `sliding` or `fixed`
+  extra: |
+    **Note:  Redis configuration values are ignored if the `cluster` strategy is used.**
```

Comment:
@GayleNeumann and I were seeing > a lot. 

I think _just_ the **Note:** should get emboldened (for lack of a better term 😄), otherwise it's too much visually. 

I'll go with your suggestion for now since that's more consistent with recent docs, and we can decide on a convention later.